database/connector.py

```diff
@@ -64,12 +70,39 @@ def insert_other_network_event(ip, port, machine_name):
     Returns:
         ObjectId(inserted_id)
     """
-    return network_events.insert_one(
+    global network_events_queue
+    network_events_queue.append(
         {
             "ip": ip,
             "port": int(port),
             "date": now(),
             "machine_name": machine_name,
             "country": str(IP2Location.get_country_short(ip).decode())
         }
-    ).inserted_id
+    )
+    return
+
+
+def insert_events_in_bulk():
+    """
+    inserts all honeypot and network events in bulk to honeypot_events and network_events collection respectively
+    """
+    global honeypot_events_queue
+    global network_events_queue
+    if honeypot_events_queue:
+        honeypot_events.insert_many(honeypot_events_queue)
```

Comment:
it would be better to share this array 
```
    honeypot_events_queue.append(
        {
            "ip": ip,
            "port": int(port),
            "module_name": module_name,
            "date": now(),
            "machine_name": machine_name,
            "country": str(IP2Location.get_country_short(ip).decode())
        }
    )
```

to

```
    honeypot_events_queue.append(
        {
            "ip": ip,
            "port": int(port),
            "module_name": module_name,
            "date": now(),
            "machine_name": machine_name,
            "country": str(IP2Location.get_country_short(ip).decode()),
            "event_type": "honeypot_event"
        }
    )
```

and them check them while inserting